Unable to build latest on fedora 28

This commit does not build for me:
```commit e9f272a49eb36d8aad77e772135f47509a2425bd (quentin/germx-hand-sanitizer-kills-99-percent-of-germs, origin/pr/2402)
Author: Quentin Young <qlyoung@cumulusnetworks.com>
Date:   Fri Jun 8 16:39:31 2018 +0000

    *: disable all sanitizers for clippy build
    
    * Move configure flag propagations out of user flags
    * Use AC_SUBST to transfer flag values to Automake
    * Set default AM_CFLAGS and AM_CPPFLAGS in common.am and change child
      Makefiles to modify these base variables
    * Add flag override to turn off all sanitizers when building clippy
    * Remove LSAN suppressions blacklist as it's no longer needed
    
    Signed-off-by: Quentin Young <qlyoung@cumulusnetworks.com>
```

bootstrap.sh run:
```
harpd@robot ~/frr> ./bootstrap.sh 
tests/Makefile.am:6: warning: AM_CPPFLAGS multiply defined in condition TRUE ...
tests/../common.am:5: ... 'AM_CPPFLAGS' previously defined here
tests/Makefile.am:1:   'tests/../common.am' included from here
sharpd@robot ~/frr> 
```

configure line:
```
sharpd@robot ~/frr> ./configure --build=x86_64-linux-gnu --prefix=/usr --includedir=\${prefix}/include --mandir=\${prefix}/share/man --infodir=\${prefix}/share/info --sysconfdir=/etc --libexecdir=\${prefix}/lib/frr --disable-maintainer-mode --enable-dependency-checking --enable-exampledir=/usr/share/doc/frr/examples/ --localstatedir=/var/log/frr/run --sbindir=/usr/lib/frr --sysconfdir=/etc/frr --enable-ospfapi=yes --enable-multipath=256 --enable-user=frr --enable-group=frr --enable-vty-group=frrvty --enable-configfile-mask=0640 --enable-logfile-mask=0640 --enable-rtadv --enable-gcc-rdynamic --with-libpam --enable-systemd=yes --enable-vtysh=yes --enable-cumulus=yes --enable-bgp-vnc=yes --with-vtysh-pager=cat --enable-datacenter=yes  --enable-dev-build=yes CC=clang --enable-fpm=yes --enable-werror=yes --enable-rpki=yes
```

Compile fails here:

```
make[2]: Entering directory '/home/sharpd/frr'
  CC       lib/lib_clippy-defun_lex.o
  CC       ospfd/ospfd.o
  CC       ospf6d/ospf6_abr.o
  CC       ospf6d/ospf6_area.o
In file included from lib/defun_lex.l:42:
In file included from /usr/include/python2.7/Python.h:8:
In file included from /usr/include/python2.7/pyconfig.h:6:
/usr/include/python2.7/pyconfig-64.h:1232:9: error: '_POSIX_C_SOURCE' macro
      redefined [-Werror,-Wmacro-redefined]
#define _POSIX_C_SOURCE 200112L
        ^
/usr/include/features.h:265:10: note: previous definition is here
# define _POSIX_C_SOURCE        200809L
         ^
In file included from lib/defun_lex.l:42:
In file included from /usr/include/python2.7/Python.h:8:
In file included from /usr/include/python2.7/pyconfig.h:6:
/usr/include/python2.7/pyconfig-64.h:1254:9: error: '_XOPEN_SOURCE' macro
      redefined [-Werror,-Wmacro-redefined]
#define _XOPEN_SOURCE 600
        ^
/usr/include/features.h:202:10: note: previous definition is here
# define _XOPEN_SOURCE  700
         ^
2 errors generated.
make[2]: *** [Makefile:3939: lib/lib_clippy-defun_lex.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/sharpd/frr'
make[1]: *** [Makefile:4421: all-recursive] Error 1
make[1]: Leaving directory '/home/sharpd/frr'
make: *** [Makefile:1959: all] Error 2
```

Reverting this commit, and things work again.

--- a/lib/subdir.am
+++ b/lib/subdir.am
@@ -236,7 +236,7 @@ lib_grammar_sandbox_LDADD = \
 	lib/libfrr.la
 
 lib_clippy_CPPFLAGS = $(AM_CPPFLAGS) -D_GNU_SOURCE -DBUILDING_CLIPPY @SAN_CLIPPY_FLAGS@
-lib_clippy_CFLAGS = $(PYTHON_CFLAGS) $(AM_CFLAGS) @SAN_CLIPPY_FLAGS@
+lib_clippy_CFLAGS = $(PYTHON_CFLAGS) @SAN_CLIPPY_FLAGS@
 lib_clippy_LDADD = $(PYTHON_LIBS)
 lib_clippy_SOURCES = \
 	lib/clippy.c \

--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -3,7 +3,7 @@ include ../common.am
 PYTHON ?= python
 
 AUTOMAKE_OPTIONS = subdir-objects
-AM_CPPFLAGS = \
+AM_CPPFLAGS += \
 	-I.. \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/lib \

